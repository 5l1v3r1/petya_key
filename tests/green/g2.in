08R8pyaitpdnhbpFAWp5DeE9EWzQ6gbvxaGUvGaXTj2ADRd9Sxse2T9GxBVsc11aE917GXSKhrALGKeSK8qi9DDUdt908fA5
