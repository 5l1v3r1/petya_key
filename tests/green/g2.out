VcwwBGdPWb22ge5b
